templates/.kokoro/build.sh

```diff
@@ -56,21 +89,30 @@ integration)
              source "${KOKORO_GFILE_DIR}/secret_manager/java-bigqueryconnection-samples-secrets"
   fi
 
-  mvn -B -pl '!google-cloud-gapic-bom,!CoverageAggregator,!java-game-servers/google-cloud-game-servers' \
-    -ntp \
-    -Penable-integration-tests \
-    -DtrimStackTrace=false \
-    -Dclirr.skip=true \
-    -Denforcer.skip=true \
-    -Dcheckstyle.skip=true \
-    -Dflatten.skip=true \
-    -Danimal.sniffer.skip=true \
-    -Djacoco.skip=true \
-    -DskipUnitTests=true \
-    -fae \
-    -T 1C \
-    verify
-  RETURN_CODE=$?
+  if [[ -n $module_list ]]
+  then
+    echo "Running Integration Tests for: ${module_list}"
+    mvn -B -pl "${module_list}" \
+        -amd \
```

Comment:
--also-make-dependencies to build the submodules. This also brings in CoverageAggregator (has no ITs)